### PR TITLE
GIX-1777: New dissolving icon

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

New icon for dissolving neuron state.

# Changes

* Upgrade gix-components
* Use new icon for dissolving state

# Tests

* Only UI changes

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
Old UI:

Screenshot 2023-07-27 at 12 06 28

New UI:

Screenshot 2023-07-27 at 12 06 16

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/utils/neuron.utils.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const stateTextMapper: StateMapper = {
   },
   [NeuronState.Dissolving]: {
     textKey: "dissolving",
-    Icon: IconPace,
+    Icon: IconDissolving,
     status: "warn",
```
Comment (dskloetd) [inline frontend/src/lib/utils/neuron.utils.ts:89]: Should this really be a warning while all the others are ok?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/utils/neuron.utils.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/utils/neuron.utils.ts:89]: Good point, the status is not yet used.

I'll remove it from the map, but in another PR.

**Pull request merged**